backend/wmg/data/schemas/cube_schema.py

```diff
@@ -22,8 +22,6 @@
     "cell_type_ontology_term_id",
     "tissue_original_ontology_term_id",
     "dataset_id",
-    "assay_ontology_term_id",
```

Comment:
removing these to shrink the cube by 66% as they are not needed by the application